build.sbt

```diff
@@ -81,9 +77,10 @@ ThisBuild / javaOptions ++= Seq(
 
 addCommandAlias(
   "compileAll",
-  "; copyResources ; scalastyle ; " +
-    "OPAL / Test / compile ; OPAL / Test / scalastyle ; " +
-    "OPAL / IntegrationTest / scalariformFormat ; OPAL / IntegrationTest / scalastyle ; OPAL / IntegrationTest / compile "
+    "OPAL / scalafmt ; OPAL / Test / scalafmt ; " +
```

Comment:
I see Test/headerCheck and IntegrationTest/headerCheck, shouldn't there also be OPAL/headerCheck?